modules/checking-mco-status-certs.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="checking-mco-status-certs_{context}"]
+= Viewing and interacting with certificates
+
+The following certificates are handled in the cluster by the Machine Config Controller (MCC) and can be found in the ControllerConfig:
+
+* `/etc/kubernetes/kubelet-ca.crt`
+* `/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem`
+* `/etc/pki/ca-trust/source/anchors/openshift-config-user-ca-bundle.crt`
+
+Use the following procedure to get information about the certificates, including the underyling bundle the certificate comes from, and the signing and subject data.
+
+.Procedure
+
+* Run the following command to get certificate information:
```

Comment:
Suggest to put what you are going to do first. Per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

```suggestion
* To get detailed certificate information, run the following command:
```

or 

```suggestion
* Get detailed certificate information by running the following command:
```